_includes/sections/working-here/managing-your-day-to-day-work.md

```diff
@@ -158,6 +158,39 @@ incurring them.
 We manage expenses using Xero. For more information about how to do this, see
 the [guide on claiming expenses](/guides/claiming-expenses)
 
+#### Travel and Accommodation Policy
+
+Currently the default for everyone in the organisation is working from home
```

Comment:
This is contradictory. I suggest:
Currently, the default for everyone in the organisation is working from home. For this period, we will consider regular travel to an office space without a business critical need as commuting.